### ISSUE TITLE
Standardize Belmont record_date field

### ISSUE THREAD

**Issue by bstarling** [Data4Democracy/town-council#14]
`record_date` output by the Belmont spider is the raw text found on website. Text should be parsed and stored as a python date object.  (This issue was recently fixed in the Dublin spider for reference)

Current output:
`record_date": "05/23/2017 6:30 PM - 7:00 PM "`

Desired output:
`"record_date": "2017-05-23"`

**Comment by josephpd3** [comment]
I was taking a look at this, and it seems like this is already being converted to a datetime object via the [parse_date_string](https://github.com/Data4Democracy/town-council/blob/master/council_crawler/council_crawler/utils.py#L9) function in `council_crawler.utils`.

Here's some of my current JSON output from `scrapy crawl belmont -o belmont.json`:

```
{
    "_type": "event",
    "ocd_division_id": "ocd-division/country:us/state:ca/place:belmont",
    "name": "Belmont, CA City Council Planning Commission Meeting",
    "scraped_datetime": "2017-07-11 20:16:28",
    "record_date": "2017-12-19 19:00:00",
    "source": "belmont",
    "source_url": "http://www.belmont.gov/city-hall/city-government/city-meetings/-toggle-all",
    "meeting_type": "Planning Commission Meeting",
    "documents": []
}
```

**Comment by bstarling** [comment]
Yup this was fixed by #17 should have been closed, sorry about that.
